soc/arm/ene_kb1200/soc.c

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2021 ENE Technology.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/init.h>
+#include <soc.h>
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(soc, CONFIG_SOC_LOG_LEVEL);
+
+#define CORTEX_M_SCR   0xE000ED10
+
+static void pmu_init()
+{
+	PMU_T *pmu = ((PMU_T *) PMU_BASE);
+	*((volatile uint32_t *)CORTEX_M_SCR)  &= (~(1L<<2));   // Disable SLEEPDEEP
+	pmu->PMUIDLE |= 0x00000001;                            // Interrupt Event Wakeup from IDLE mode Enable
+	pmu->PMUSTOP |= PMU_STOP_WU_GPTD;                      // GPTD wake up from STOP mode enable.
+	pmu->PMUSTOP |= (PMU_STOP_WU_EDI32 | PMU_STOP_WU_SWD); // SWD EDI32 wake up from STOP mode enable
+}
+
+#define KB1200_CLKCFG 0x40000020
+
+static void clock_init()
+{
+	if (CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC == 96000000)
```

Comment:
not a fan of these if-conditionals without braces

Reply:
It's even stated in guidelines : https://docs.zephyrproject.org/latest/contribute/guidelines.html#coding-style.
As @ycsin said, please run clang-format on edited files following : https://docs.zephyrproject.org/latest/contribute/guidelines.html#clang-format